system/common.xml

```diff
@@ -37,6 +37,13 @@
 		<isset property="env.STF_BRANCH"/>
 	</condition>
 	
+	<condition property="svttest_repo" value="${env.VENDOR_TEST_REPOS}" else="Please mention SVTTestRepo in VENDOR_TEST_REPOS field">
+		<isset property="env.VENDOR_TEST_REPOS"/>
+	</condition>
+	<condition property="svttest_repo_branch" value="${env.VENDOR_TEST_BRANCHES}" else="master">
+		<isset property="env.VENDOR_TEST_BRANCHES"/>
+	</condition>
```

Comment:
`VENDOR_TEST_REPOS` and `VENDOR_TEST_BRANCHES` could be multiple repos and branches. And they may not be system test repo. I do not think cannot simply set it this way.

Reply:
Agree, I'll find a way to get property set for only SVTTestRepo. Thank you.